src/dunes/__main__.py

```diff
@@ -80,7 +80,7 @@ def load_all_modules_from_dir(plugin_dir):
 
     handle_simple_args()
 
-    dune_sys = dune(args)
+    dune_sys = dunes(args)
 
     for module in modules:
         if hasattr(module, 'set_dune'):
```

Comment:
I'm not sure what to do here, but a change is needed.

```python
    for module in modules:
        if hasattr(module, 'set_dunes'):
            module.set_dunes(dune_sys)
        elif hasattr(module, 'set_dune'):
            # prefer new `set_dunes`, so only test for `set_dune` if above does not exist.
            # probably need to warn the user this may not work? <-------------------- what to do here?
            module.set_dune(dune_sys)
```

Reply:
> There is no any third party plugin so I just renamed to set_dunes as in the plugin examples. 

We might theorize there are no third party plugins, but do we really know? 

The following lines of code are inexpensive and would really help someone that's already added a plugin: 
```python
        # Warn users of outdated plugins.
	elif hasattr(module, 'set_dune'):
            print( f"WARNING: {module} uses old interface." )
```
I suppose the _probability_ of a 3rd party plugin is low enough to make this a nice to have bit of code that doesn't really add much overall. Feel free to either add the warning, or ignore and resolve this conversation as you like.